src/Microsoft.Win32.SystemEvents/tests/SystemEventsTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+using static Interop;
+
+namespace Microsoft.Win32.SystemEventsTests
+{
+    public abstract class SystemEventsTest
+    {
+        IntPtr s_hwnd = IntPtr.Zero;
+
+        public const int PostMessageWait = 100;
```

Comment:
as mentioned above, timing official runs varies substantially. the only purpose of this wait is to try to prevent one hung test taking out all the others in the class. Since we're dealing wit ha 20 minute timeout in Helix and these tests probably take a minute or so you could set this to 1 minute and still probably achieve that goal even if every one of these hung.

Reply:
Looks like you may hae hit that even in CI here.

Reply:
Yeah, I suspected that.  I figured I'd start by aiming low to keep things fast in the case of a failure but I guess the I'll need to dial it back.